packages/graph/hash_graph/lib/graph/src/store/postgres/query/condition.rs

```diff
@@ -142,19 +100,22 @@ mod tests {
         rendered: &'static str,
         parameters: &[&'q dyn ToSql],
     ) {
-        let mut parameter_list = Vec::new();
-        let condition = Condition::from_filter(filter, &mut parameter_list, None);
+        let mut compiler = SelectCompiler::new();
```

Comment:
Not something blocking, but wondering why the tests for `Condition` here makes use of the `SelectCompiler`? Couldn't we use the previous setup using the transpile directly? I guess using `Condition` makes these feel more like unit tests, but I don't feel strongly about it.

I might be missing something :)

Reply:
Previously it was possible to compile an `Expression` by only injecting some values, but the number of things, which needs to be passed to `Expression::from_filter` got unmaintainable, so I moved the function to `SelectCompiler` instead.

Reply:
Okay, fair enough :+1: